src/backend/marsha/core/tests/test_api_thumbnail.py

```diff
@@ -440,3 +470,58 @@ def test_api_thumbnail_initiate_upload_instructor_read_only(self):
         )
 
         self.assertEqual(response.status_code, 403)
+
+    @override_settings(THUMBNAIL_SOURCE_MAX_SIZE=10)
+    def test_api_thumbnail_initiate_upload_file_too_large(self):
+        """Instructor users should not be allowed to initiate an upload."""
+        video = VideoFactory(
+            id="c10b79b6-9ecc-4aba-bf9d-5aab4765fd40", upload_state="ready"
```

Comment:
Most of the time you don't need to provide an id, you should let the factory generates a random one for you and if you need it in your test you should then use `video.pk`